LICENSE

```diff
@@ -1,6 +1,6 @@
 Copyright (c) 2021
 
-Ross Viljoen
+The JuliaGaussianProcess Contributors
```

Comment:
Didn't check how it's done in all the other packages, but particularly when putting packages into an org I find it much more inviting to future contributors if it just says "Copyright the people who contributed to it"... let me know your thoughts on this, happy to discuss it more in depth

Reply:
I think that for the license we should just have "The JuliaGaussianProcesses organization" and for the author field in the `Project.toml` have `JuliaGaussianProcesses and contributors"

Reply:
My inclination has normally been to go with "Person who created the package + the org", so "Ross Viljoen and the JuliaGaussianProcess org", so that the person who did the initial work retains some credit -- in this case, it would be good to keep Ross' name on it imho. Just a suggestion though.

I've just taken a look through the other packages in the org, and it seems like we're pretty inconsistent. AbstractGPs just mentions JuliaGPs, KF mentions Theo and Turing. We should probably agree to a consistent approach and stick to it, but this PR isn't the place for that.

Consequently, could I suggest that we leave the license as-is in this PR, and update it elsewhere once we've made an org-wide decision?

Reply:
Yeah this pr isn't gonna get merged anyways: ) just used the opportunity to spark the discussion.

I'd prefer _just_ "the org"; I think it makes more sense from an overall point of view to use this package, for example, to bundle SVGP, Laplace, EP - and I believe we incentivise contributions by new people by not giving the feeling that you're just increasing someone else's credit with your contribution.

From my personal point of view, as much as I want to get myself not to care about attribution as much, I can feel it inside myself that I'd rather create github.com/st--/LaplaceGPs.jl or something like that that I can then put _my_ name on than to add a large set of functionality to "someone else's project". Whereas if it's just listed as "the org" then it feels like by contributing more code I deserve more to be part of the org and take my share of the credit from there. Which is why I changed the text in my branch before pushing it.